documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -58,15 +60,18 @@ EOF
 <1> The `ownerReferences` section is optional.
 <2> Specify the type of source provider. Allowed values are `ovirt`, `vsphere`, `openstack`, `ova`, and `openshift`. This label is needed to verify the credentials are correct when the remote system is accessible and, for {rhv-short}, to retrieve the {manager} CA certificate when a third-party certificate is specified.
 <3> The `stringData` section for OVA is different and is described in a note that follows the description of the `Secret` manifest.
-<4> Specify the vCenter user, the {rhv-short} {manager} user, or the {osp} user.
+<4> Specify the vCenter user, the {rhv-short} {manager} user, or the {osp} user. For {ocp-short}, instead of entering a `user`, create a service account and a token, as described in xref:adding-destination-providers[Adding an OpenShift Virtualization destination provider]. This property is not used for OVA.
 <5> Specify the user password.
 <6> Specify `<true>` to skip certificate verification, which proceeds with an insecure migration and then the certificate is not required. Insecure migration means that the transferred data is sent over an insecure connection and potentially sensitive data could be exposed. Specifying `<false>` verifies the certificate.
 <7> {osp} only: Specify the domain name.
 <8> {osp} only: Specify the project name.
 <9> {osp} only: Specify the name of the {osp} region.
 <10> {rhv-short} and {osp} only: For {rhv-short}, enter the {manager} CA certificate unless it was replaced by a third-party certificate, in which case, enter the {manager} Apache CA certificate. You can retrieve the {manager} CA certificate at https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA. For {osp}, enter the CA certificate for connecting to the source environment. The certificate is not used when `insecureSkipVerify` is set to `<true>`.
 <11> Specify the API end point URL, for example, `https://<vCenter_host>/sdk` for vSphere, `https://<engine_host>/ovirt-engine/api` for {rhv-short}, or `https://<identity_service>/v3` for {osp}.
-<12> {ocp-short} only:  Token for a service account with `cluster-admin` privileges.
+This property is not used for OVA.
+<12> VMware only: Specify the vCenter SHA-1 fingerprint.
+<13> {ocp-short} only:  Token for a service account with `cluster-admin` privileges.
+>>>>>>> cf12e63e (OVA and OCP in NetworkMap and StorageMap)
```

Comment:
something went wrong with the merge-conflict resolution, I wouldn't expect to see the SHA-1 fingerprint again..